plugins/inputs/win_perf_counters/win_perf_counters_test.go

```diff
@@ -58,31 +58,31 @@ func (m *FakePerformanceQuery) Open() error {
 
 func (m *FakePerformanceQuery) Close() error {
 	if !m.openCalled {
-		return errors.New("CloSe: uninitialized query")
+		return errors.New("in Close: uninitialized query")
```

Comment:
is adding all of these "in" prefix resolving a linter item?

Reply:
@srebhan asked me to do it https://github.com/influxdata/telegraf/pull/13057#discussion_r1162594837 so I changed these messages in whole file (to be consistent).

Reply:
ah thanks for the background